en/docs/develop/create-integration-project.md

```diff
@@ -1,11 +1,15 @@
-# Creating an Integration Project
+# Create an Integration Project
 
-1. Launch Visual Studio Code with the Micro Integrator extension installed.
+An integration project in WSO2 Micro Integrator (MI) is a structured collection of artifacts and configurations designed to facilitate seamless communication and data exchange between diverse systems, applications, and services.
```

Comment:
Do we add configurations ?

Reply:
Fixed in https://github.com/wso2/docs-mi/pull/1151/commits/d0906719454a3f5a2629d336c1192aeb6912502e.